src/OpenTelemetry.Exporter.InMemory/InMemoryExporter.cs

```diff
@@ -23,6 +24,7 @@ public class InMemoryExporter<T> : BaseExporter<T>
     {
         private readonly ICollection<T> exportedItems;
         private readonly ExportFunc onExport;
+        private bool disposed = false;
```

Comment:
nit: Technically you don't need the `= false` ([CA1805](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1805))

Reply:
Ahh IIRC this isn't the first time I missed it (habit from C++).

Reply:
Fixed, and also fixed the PrometheusExporter.